### PR TITLE
fixed the signup form submission

### LINKED ISSUE #9: [FEAT] : sign up
## Describe

Inside the Sign Up page the Form submission was not done 
Screenshot 2024-05-11 174848

### PR DESCRIPTION
**Pull request by RahulK4102**
## Describe
the data which we were collecting from the user on sign up page was not getting stored in database

## Issue
resolves #9

### CODE REVIEW

File: database/evaluation_db.sql [hunk 1]
```diff
@@ -166,6 +165,37 @@ INSERT INTO `faculty_list` (`id`, `school_id`, `firstname`, `lastname`, `email`,
...
+  `avatar` blob NOT NULL,
+  `date_created` datetime NOT NULL
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_general_ci;
+
+--
+-- Dumping data for table `login`
+--
+
+INSERT INTO `login` (`school_id`, `firstname`, `lastname`, `email`, `password`, `class_id`, `avatar`, `date_created`) VALUES
+(2147483647, 'First Name', 'Last Name', 'Email', '$2y$10$V92q2fOjeVF2w3nytZW.0ujYaXRIkgGjvgxM4YeORjH.DYoeghPgG', 0, 0x6e6f2d696d6167652d617661696c61626c652e706e67, '2024-05-11 18:07:06'),
```
Comment (aswinikalyan30) [inline database/evaluation_db.sql:188]: Is this SQL exported from the local db setup? 
I suggest you can remove the repetitive records here

**Review by aswinikalyan30 — COMMENTED** [review]
I think it would be better if you revert the changes for the SQL file

File: database/evaluation_db.sql (hunk 1 above)
Comment (RahulK4102) [reply database/evaluation_db.sql]: I have removed the SQL which was exported from my local db setup

**Review by aswinikalyan30 — APPROVED** [review]
Thanks!

**Pull request merged**